src/settings.js

```diff
@@ -1393,6 +1399,19 @@ var MINIFY_HTML = 1;
 // is received from wasm-emscripten-finalize, which reads it from the features section.
 var BINARYEN_FEATURES = [];
 
+// the minimum/maximum targeted GL feature level.
+// The concept of "feature level" is coined to workaround some versioning
+// difficulties, using the following mappings:
+//
+//   | GL feature level | WebGL version | OpenGL ES version |
+//   | ---------------- | ------------- | ----------------- |
+//   | 10               | 1.0           | 2.0               |
+//   | 20               | 2.0           | 3.0               |
+//   | 30               | 2.0 Compute   | 3.1               |
+//
+var GL_MIN_FEATURE_LEVEL = 10;
```

Comment:
why is the minimum feature level needed? i feel i don't understand its purpose yet.

Reply:
Just to be symmetric with `GL_MAX_FEATURE_LEVEL`, but not used anywhere.

Reply:
Might be less confusing to add in a comment that the MIN part is for future optimization opportunities or something like that?

Reply:
I think it would be best to not include it at all. If a future patch wants it, they can add it.

Reply:
@hujiajie , can you remove this?